packages/web/src/components/createCell.tsx

```diff
@@ -128,7 +129,13 @@ export function createCell<CellProps = any>({
         {({ error, loading, data, ...queryRest }) => {
           if (error) {
             if (Failure) {
-              return <Failure error={error} {...queryRest} {...props} />
+              return (
+                <Failure
+                  error={error}
+                  {...{ updating: loading, ...queryRest }}
+                  {...props}
```

Comment:
Maybe something like this? Seems a bit weird to spread for 1, but not the other? 🤷‍♂️
```suggestion
                  {...{ updating: loading, ...queryRest, ...props }}
```

Reply:
yeah I just copied what was done below lol. This does look better

Reply:
I'll clean this up the the cell typing PR!